src/docs/development/ui/assets-and-images.md

```diff
@@ -340,6 +340,20 @@ NSString* path = [[NSBundle mainBundle] pathForResource:key ofType:nil];
 For a more complete example, see the implementation of the
 Flutter [video_player plugin][].
 
+The plugin [ios_platform_images][] on pub.dev wraps up this logic in a
+convenient category.  It allows writing:
+
```

Comment:
use {% samplecode %} tags and add swift too.

Reply:
Added swift example, no one else is using the samplecode tag.